src/commands/project/pipeline.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/*eslint semi: "error"*/
```

Comment:
we have attempting to remove page wide eslint-disable, please do not add them. Please contact @evanlouie or/and @bnookala if we want to semi: error in .eslintrc thanks

Reply:
Will remove!